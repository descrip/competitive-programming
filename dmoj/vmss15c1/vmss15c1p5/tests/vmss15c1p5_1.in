3 250 250
gala 500 20 4
goldendelicious 450 1 25
green 380 13 4
