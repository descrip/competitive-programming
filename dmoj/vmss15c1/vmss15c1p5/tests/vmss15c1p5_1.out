10110
gala 1
goldendelicious 7
green 17
